### DISCUSSION TITLE
How can I use cotnainer-queries (@sm / @lg, etc) inside postCSS or SCSS?

### DISCUSSION THREAD

**Discussion by demiro** [tailwindlabs/tailwindcss#12122] [Help]
So I would like to use container queries inside my SCSS the same way I am using media queries (`@screen md{ ...}`) 

but neither `@md{...}` nor `@container md{...}` seem to be working

**Comment by wongjn** [comment]
You could consider using `theme()` like:

```scss
@container (min-width: theme(container.md)) {
  /* … */
}
```

Alternatively, you could use `@apply` like:

```scss
selector {
  @apply @md:text-black;
}
```
